File: 1018-largest-perimeter-triangle/largest-perimeter-triangle.ts
function largestPerimeter(nums: number[]): number {
   nums.sort((a,b)=> a-b);

   for(let i=nums.length-1;i>1;i--){
         let a = nums[i];
         let b = nums[i-1];
         let c = nums[i-2];

         if(a < b+c) return a+b+c;
   }
   return 0;
};

File: 1018-largest-perimeter-triangle/largest-perimeter-triangle.cpp
class Solution {
public:
    int largestPerimeter(vector<int>& nums) {
        sort(nums.begin(), nums.end());
        int n = nums.size() - 1;
        int ans = 0;

        for (int i = n; i > 1; i--) {
            int a = nums[i];
            int b = nums[i-1];
            int c = nums[i-2];

            if(a < b+c){
                return a+b+c;
            }
        }
        return ans;
    }
};